app/views/cloud_object_store_container/new.html.haml

```diff
@@ -0,0 +1,62 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "cloudObjectStoreContainerFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Basic Information')
+  .form-horizontal
+    .form-group{"ng-class" => "{'has-error': angularForm.storage_manager_id.$invalid}"}
+      %label.col-md-2.control-label
+        = _('Storage Manager')
+      .col-md-8
+        = select_tag("storage_manager_id",
+          options_for_select([["<#{_('Choose')}>", nil]] + @storage_manager_choices.sort, disabled: ["<#{_('Choose')}>", nil]),
```

Comment:
@miha-plesko if you don't need all the attributes, I suggest you limit the list to those that are important, e.g. add `&attributes=id,name` to the request to get just the name and id:

```json
{
  "name": "providers",
  "count": 8,
  "subcount": 2,
  "resources": [
    {
      "href": "http://localhost:3000/api/providers/17",
      "id": 17,
      "name": "OS XLAB Cinder Manager"
    },
    {
      "href": "http://localhost:3000/api/providers/23",
      "id": 23,
      "name": "AWS XLAB EBS Storage Manager"
    }
  ],
  "actions": [
    {
      "name": "query",
      "method": "post",
      "href": "http://localhost:3000/api/providers"
    },
    {
      "name": "create",
      "method": "post",
      "href": "http://localhost:3000/api/providers"
    },
    {
      "name": "edit",
      "method": "post",
      "href": "http://localhost:3000/api/providers"
    },
    {
      "name": "refresh",
      "method": "post",
      "href": "http://localhost:3000/api/providers"
    },
    {
      "name": "delete",
      "method": "post",
      "href": "http://localhost:3000/api/providers"
    }
  ]
}
```